### ISSUE TITLE
resume management software for solo headhunter

### ISSUE THREAD

**Issue by Golamoka** [Gunrh/Automated-Resume-Screening-and-Analysis#1]
Dear sir, Is this software suitable for personal headhunter to management, quick filtering, notes and keywording job seekers' resumes?

**Comment by Gunrh** [comment]
Yes, the software is adapted for headhunters for quick  filtering screening of resumes.

This is the basic version, there is a more accurate version based on deep learning algorithms. that is offered for sale
